src/app/component/SearchUser.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState, Fragment } from "react";
+import { Box, TextField, Avatar } from "@material-ui/core";
+import Autocomplete from "@material-ui/lab/Autocomplete";
+import CircularProgress from "@material-ui/core/CircularProgress";
+import { makeStyles } from "@material-ui/core/styles";
+import _ from "lodash";
+
+import { Users } from "../model";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    paddingBottom: theme.spacing(2),
+  },
+  avatar: {
+    marginRight: theme.spacing(1),
+  },
+  textField: {
+    paddingRight: "9px!important",
```

Comment:
Can we not use !important ? This might cause potential UI bugs in the future.

Reply:
+1 for this. Strongly believe !important should never be used. If our styles are set up correctly, we should not need to force specificity

Reply:
Understood, will remove important!